eosdis-nasa/earthdata-pub-api
# Adds initial workflow for PO DAAC

# Description

Adds an initial workflow for PO DAAC and makes PO DAAC an active DAAC

## Spec

See Ticket: https://bugs.earthdata.nasa.gov/browse/EDPUB-1191

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.
3. Deploy the application locally
4. Create a New Request and verify that PO DAAC is an active DAAC
5. Click on the PO DAAC radio button and submit
6. Verify that the workflow name is "PO DAAC Default Workflow"
7. Continue through the forms and and approvals
8. Verify that the export metadata step occurs after the Data Publication Request Form Review
9. Continue and verify that the form Closes.
